orbit/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## Orbit 1.32.0 (Aug 29, 2024)
+
+* bumped macadmins extension to use SOFA feed sofafeed.macadmins.io
+
+In Fleet Desktop, My Device status is now refreshed when user clicks on My Device or Self-Service dropdown option.
+
+* Updated go to go1.22.6
+
+* Adds ability for MDM migrations if the host is manually enrolled to a 3rd party MDM.
```

Comment:
```suggestion
* Bumped macadmins extension to use SOFA feed sofafeed.macadmins.io

* In Fleet Desktop, My Device status is now refreshed when user clicks on My Device or Self-Service dropdown option.

* Updated go to go1.22.6

* Added ability for MDM migrations if the host is manually enrolled to a 3rd party MDM.
```

Reply:
Thanks! And I'll remove the item that you already released in the patch. I'm not pushing the change yet because it will cancel the ongoing builds :)

Reply:
oohh, I forgot about that too, thanks!